ui/src/app/base/components/DhcpForm/DhcpForm.tsx

```diff
@@ -40,7 +42,9 @@ type Props = {
   analyticsCategory: string;
   id?: DHCPSnippet["id"];
   onSave?: () => void;
-} & Partial<FormikFormProps<DHCPFormValues>>;
+} & Partial<
+  FormikFormContentProps<DHCPFormValues, FormikErrors<DHCPFormValues>>
```

Comment:
I wonder if this repetition is necessary. Have you tried setting `FormikErrors<V>` as default for the second type parameter in `FormikFormContentProps`?